jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -198,7 +198,7 @@ YUI.add('user-profile', function() {
           model.owner = model.ownerTag;
           return model;
         });
-      } else {
+      } else if (data.map) {
```

Comment:
After checking into this more, I decided to keep `data.map` in place. If `data` is an empty list, we still want the mapping to happen so that `modelList` ends up being assigned to an empty list. Just checking to see if `data` is truthy doesn't make sense, as we've already checked `data.length` by that point. So I'm sticking with the original code, which verifies that the `map` function is present before calling it.